src/main/proto/th2_grpc_common/common.proto

```diff
@@ -300,3 +300,16 @@ message EventBatch {
   EventID parent_event_id = 1;
   repeated Event events = 2; // Events optional related to between themselves. No events outside this batch should refer to the events in this batch.
 }
+
+message Notification {
+  string name = 1;
+  string description = 2;
+  NotificationKind kind = 3;
+}
```

Comment:
Let's use NotificationEvent as a common message wrapper and separate messages for each kind of notification.
For example
message NotificationEvent {
  oneof test_oneof {
    BookPageNotification book_page = 1;
  }
}

message BookPageNotification {
  // it can be empty for the first version 
}

Reply:
1. Is `test_oneof` correct name to use?
2. Should we keep only one kind `BookPageNotification` or maybe two: `BookNotification`/`PageNotification`?